web/themes/custom/novel/templates/paragraphs/paragraph--hero.html.twig

```diff
@@ -0,0 +1,22 @@
+<section class="hero">
+    <a href="{{ content.field_hero_link.0["#url"] }}" class="hero-content arrow arrow__hover--right-large">
+        {{ content.field_hero_categories }}
+        <div class="hero-date">
+            {% if content.field_hero_content_type.0 %}
+                <span class="hero-date__text">{{ content.field_hero_content_type }}</span>|
+            {% endif %}
+            <time class="hero-date__date">{{ content.field_hero_date }}</time>
+        </div>
+        <h1 class="hero-title">{{ content.field_hero_title }}</h1>
```

Comment:
is hero title always set? otherwise, it'll result in an empty `<h1>` here, which is not good